docs/feature_flag.md

```diff
@@ -22,7 +22,7 @@ In the example above, we have defined a `String` type feature flag named `foo` a
 It has variants of `bar` and `baz`, referring to respected values of `BAR` and `BAZ`.
 The default variant is set to`bar`.
 
-## featureFlagSpec
+## flagSpec
 
-The `featureFlagSpec` is an object representing the flag configurations themselves.
+The `flagSpec` is an object representing the flag configurations themselves.
```

Comment:
```suggestion
The `flagSpec` is an object representing the flag definitions themselves.
```

I think "definitions" is better here. It's the term we use for this generally in OpenFeature, and we already use "configuration" so much